src/app/app.component.js

```diff
@@ -30,15 +32,40 @@ function AppComponentCtrl(
 
     this.settings = dimSettingsService;
     $scope.$watch(() => this.settings.itemSize, (size) => {
-      document.querySelector('html').style.setProperty("--item-size", `${size}px`);
+      if (!this.isPhonePortrait) {
+        document.querySelector('html').style.setProperty("--item-size", `${size}px`);
+      }
     });
     $scope.$watch(() => this.settings.charCol, (cols) => {
-      document.querySelector('html').style.setProperty("--character-columns", cols);
+      if (!this.isPhonePortrait) {
+        document.querySelector('html').style.setProperty("--character-columns", cols);
+      }
     });
     $scope.$watch(() => this.settings.vaultMaxCol, (cols) => {
       document.querySelector('html').style.setProperty("--vault-max-columns", cols);
     });
 
+    const sizesMobile = {
+      3: 64,
+      4: 51,
+      5: 42
+    };
```

Comment:
I don't think we should hardcode sizes. We should instead define layouts that calculate the image size using the "vw" CSS units: https://css-tricks.com/viewport-sized-typography/

Reply:
I hardcoded because we already have many sizes hardcoded.  Also, I did in a way that would use the css-vars we have now instead of creating new ones, something that I believe you would frown upon.

Reply:
Right, but I don't think we should hardcode these sizes. Phone sizes vary, and we want a consistent "fill" across the width. So we want something more like `calc((100vw - 80px) / --character-cols)` - it'll adjust the sizing of items automatically based on screen size. These sizes may be OK for some phones but may overflow others.